ui/components/multichain/app-header/app-header.js

```diff
@@ -328,20 +346,20 @@ export const AppHeader = ({ location }) => {
                     justifyContent={JustifyContent.flexEnd}
                     width={BlockSize.Full}
                   >
+                    {isMetamaskNotificationsEnabled &&
+                      !accountOptionsMenuOpen && (
+                        <Box
+                          style={{ position: 'relative' }}
```

Comment:
Why inline style?

I'm fine with using inline vs classname, but just want to make sure.

Reply:
The style I understand - the counter is absolute, so we want to contain it within this parent.

Reply:
Other places do the same, so this is fine.